src/cli/abrtcli/cli/list.py

```diff
@@ -0,0 +1,34 @@
+from abrtcli import cli, config
+from abrtcli.i18n import _
+from abrtcli.match import match_lookup
+from abrtcli.utils import format_problems, sort_problems
```

Comment:
Unused import `sort_problems`.

Reply:
Not sure why that is, because it’s absolutely used.

Reply:
Oh, `sort_problems`. Me blind.